ui/DesignSystem/Component/Wallet/SendReceive.svelte

```diff
@@ -0,0 +1,114 @@
+<script>
+  import { Button, Input, Text, Title } from "../../Primitive";
+  import Copyable from "../Copyable.svelte";
+  import QR from "../QR.svelte";
+
+  $: currentlyActive = "send";
```

Comment:
Instead of using string literals, we could make a typescript `enum` in a new `wallet.ts` file:

```
export enum WalletState = {
  Send = "send",
  Receive = "receive"
}
```
and then reference it here like `WalletState.Send` (or something like this; not sure what the eventual states will be).

Reply:
Does that make sense if this is just a local thing to this component. Like nothing else needs this.

Reply:
True, but I like to avoid [magic strings](https://softwareengineering.stackexchange.com/questions/365339/what-is-wrong-with-magic-strings) as much as possible. If we use `enums` instead of magic strings, we can see a lot of silly errors at compile time instead of waiting for the application to run and not behave as expected.

If you'd rather do this in JavaScript, [here](https://www.sohamkamani.com/blog/2017/08/21/enums-in-javascript/) is a good blog post on faking JS enums. Since we do have TypeScript though, we might as well use it.

Reply:
Alternatively since there are only 2 states, a `boolean` might make more sense than a `string`

Reply:
Makes sense. https://github.com/radicle-dev/radicle-upstream/pull/662/commits/e49c414d26ad008660f5215e85a1838f27d37f3c